src/schema/v2/partner.ts

```diff
@@ -482,6 +484,50 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
         type: GraphQLString,
         resolve: ({ default_profile_id }) => default_profile_id,
       },
+      documentsConnection: {
+        description: "A connection of documents at a partner.",
+        type: partnerDocumentsConnection.connectionType,
+        args: pageable({
+          partnerID: {
+            type: new GraphQLNonNull(GraphQLString),
+            description: "The slug or ID of the Partner",
+          },
+          page: {
+            type: GraphQLInt,
+          },
+          size: {
+            type: GraphQLInt,
+          },
```

Comment:
**suggestion**: I think our connection type helpers expose `first/last/after/before` arguments for you. `page/size` can be used when translating the request for Gravity's API but I don't think they're normally exposed here as query field arguments.